crates/net/network/src/transactions/fetcher.rs

```diff
@@ -36,42 +37,99 @@ const MARGINAL_FALLBACK_PEERS_PER_TX: u8 = 1;
 /// an announcement after it has been ejected from the hash buffer.
 const MAX_REQUEST_RETRIES_PER_TX_HASH: u8 = 2;
 
-/// Maximum concurrent [`GetPooledTxRequest`]s.
-const MAX_CONCURRENT_TX_REQUESTS: u32 = 10000;
+/// Default maximum concurrent [`GetPooledTxRequest`]s.
+pub(super) const DEFAULT_MAX_CONCURRENT_TX_REQUESTS: u32 = 10000;
+
+/// Maximum concurrent [`GetPooledTxRequest`]s to allow per peer.
+pub(super) const MAX_CONCURRENT_TX_REQUESTS_PER_PEER: u8 = 1;
```

Comment:
Is this not too low given the default max peer count is around 100?

Reply:
thanks for the input. then rather `DEFAULT_MAX_CONCURRENT_TX_REQUESTS` is too high and should also default to 100.